.github/workflows/create-release.yml

```
@@ -27,6 +27,8 @@ jobs:
           ref: develop
       - name: Bump version
         run: node ./scripts/bump-version.js "${{ needs.release-pr.outputs.new-version }}"
+      - name: Update CDN version in README.md
+        run: node ./scripts/bump-cdn-version.js
```

The sripts `bump-version.js` and `bump-cdn-version.js` can be grouped in a `bump` directory after #5546. Additionally, I feel that the name of this script is really confusing, I would vote for:

```suggestion
        run: node ./scripts/bump/readme-cdn-urls-version.js
```

---

> The sripts bump-version.js and bump-cdn-version.js can be grouped in a bump directory after #5546. 

Not sure if "bump" is a particularly descriptive folder name :thinking: 


> Additionally, I feel that the name of this script is really confusing, I would vote for:

I agree couldn't come up with something better when I created the file and forgot to check before submitting this PR. I like including "urls", but specifying the README seems a bit overkill (plus, who knows where else we might want to update these URLs in the future). How about `update-cdn-urls.js`?

---

> Not sure if "bump" is a particularly descriptive folder name

What about `versioning` or just `version`?

> I like including "urls", but specifying the README seems a bit overkill (plus, who knows where else we might want to update these URLs in the future). How about `update-cdn-urls.js`?

I agree, that would be also descriptive!

---

> What about versioning or just version?

Hmm, I'm not convinced. I'm in favor of creating a folder for this but if the name is bad I'd rather hold off on it or now :thinking:

`release` is another option, what do you think about that?

In any case, I don't think this topic is blocking for this PR - or should even be solved in this PR. If anything, this PR can be merged and the folder name can be resolved in #5546.

I'd definitely say we should merge this in before the [v5 release](https://github.com/simple-icons/simple-icons/milestone/4), so to be sure we don't forget about this I added it to the milestone.